corehq/apps/linked_domain/tests/test_update_previews.py

```diff
@@ -0,0 +1,43 @@
+from unittest import mock
+
+from corehq.apps.linked_domain.local_accessors import get_enabled_previews
+from corehq.apps.linked_domain.tests.test_linked_apps import BaseLinkedAppsTest
+from corehq.apps.linked_domain.updates import update_previews
+from toggle.shortcuts import set_toggle
+
+from corehq.feature_previews import FeaturePreview
+from corehq.toggles import NAMESPACE_DOMAIN
+
+TEST_PREVIEW = FeaturePreview(
+    slug='test_preview',
+    label='Test label',
+    description='Test description',
+)
+
+
+@mock.patch('corehq.feature_previews.all_previews_by_name', mock.Mock(return_value={'TEST_PREVIEW': TEST_PREVIEW}))
```

Comment:
I did this for feature flags as well. I agree that it would be better to avoid mocks, but I also didn't want to depend on a random production feature flag or preview in the event that it changes in the future. For example, I don't like the idea of removing a feature preview resulting in this test failing. 

Ideally I could have created a toggle and registered it in a way that `all_previews_by_name` finds it, but was unable to do so. If you have any ideas I'm open to hearing them.